src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="star stars--0">
```

Comment:
```suggestion
    <div class="stars stars--0">
```
В тебе є елементи stars__star та модифікатори stars--1, отже блок має називатись stars